Jenkinsfile

```diff
@@ -16,16 +16,16 @@ pipeline {
         // We have to point CMake back at g++-5 for now.
         sh '''
 export CC=gcc-5 CXX=g++-5
-cargo build --features native -vv --color never 2>&1 | grep --line-buffered -v '     Running `rustc --' | grep --line-buffered -v '       Fresh'
-cargo test --features native
+cargo build -vv --color never 2>&1 | grep --line-buffered -v '     Running `rustc --' | grep --line-buffered -v '       Fresh'
```

Comment:
This file is very old and not required anymore. You can remove it entirely :slightly_smiling_face: